README.md

```diff
@@ -4,6 +4,12 @@ Vita-Min is a Rails app that helps people access the VITA program through a digi
 
 ## Setup
 
+Quick start script
+
+```bash
+bin/setup
+```
+
```

Comment:
Should we mention somehow that if you run this script you will not need to perform the following steps?